File: Python/1985.py
def entrada():
    n = int(input())
    x = list()
    for i in range(n):
        a = input().split()
        x.append(a)
    return x

def preco(x):
    comanda = 0
    for i in range(len(x)):
        if x[i][0] == '1001':
            comanda += int(x[i][1]) * 1.50
        elif x[i][0] == '1002':
            comanda += int(x[i][1]) * 2.50
        elif x[i][0] == '1003':
            comanda += int(x[i][1]) * 3.50
        elif x[i][0] == '1004':
            comanda += int(x[i][1]) * 4.50
        elif x[i][0] == '1005':
            comanda += int(x[i][1]) * 5.50

    return comanda

def main():
    print('{:.2f}'.format(preco(entrada())))

main()
        

File: CSharp/1985.cs
using System;
using System.Collections.Generic;

class URI
{
    static void Main()
    {
        Console.WriteLine("{0:F2}", Preco(Entrada()));
    }

    static List<List<string>> Entrada()
    {
        int n = int.Parse(Console.ReadLine());
        var x = new List<List<string>>();
        
        for (int i = 0; i < n; i++)
        {
            var a = new List<string>(Console.ReadLine().Split());
            x.Add(a);
        }

        return x;
    }

    static double Preco(List<List<string>> x)
    {
        double comanda = 0;
        
        foreach (var item in x)
        {
            string codigo = item[0];
            int quantidade = int.Parse(item[1]);

            switch (codigo)
            {
                case "1001":
                    comanda += quantidade * 1.50;
                    break;
                case "1002":
                    comanda += quantidade * 2.50;
                    break;
                case "1003":
                    comanda += quantidade * 3.50;
                    break;
                case "1004":
                    comanda += quantidade * 4.50;
                    break;
                case "1005":
                    comanda += quantidade * 5.50;
                    break;
            }
        }

        return comanda;
    }
}



/*

Pode ser necessário ajustar a cultura para que imprima corretamente com "." em sua máquina:

using System.Globalization;

...

CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
*/